masterlist.yaml

```diff
@@ -6,6 +6,9 @@ common:
   - &SkyUI
     name: 'SkyUI_SE.esp'
     display: '[SkyUI](https://www.nexusmods.com/skyrimspecialedition/mods/12604)'
+  - &FNIS
+    name: 'tools/GenerateFNIS_for_Users/GenerateFNISforUsers.exe'
+    display: '[Fores New Idles in Skyrim SE](https://www.nexusmods.com/skyrimspecialedition/mods/3038)'
```

Comment:
I had initially planned to use it with [XP32 Maximum Skeleton Special Extended](https://www.nexusmods.com/skyrimspecialedition/mods/1988) but one of its options does not require FNIS to function.
So I had to switch from using only:
```yaml
    req:
      - name: *FNIS
```
to a message instead:
```yaml
    msg:
      - type: error
        content: 'You have installed [XP32 Maximum Skeleton Special Extended](https://www.nexusmods.com/skyrimspecialedition/mods/1988/) features which require [Fores New Idles in Skyrim SE](https://www.nexusmods.com/skyrimspecialedition/mods/3038) to function correctly, but it is not installed.'
        condition: 'file("meshes/actors/character/character assets/skeleton.hkx") and not file("tools/GenerateFNIS_for_Users/GenerateFNISforUsers.exe")'
```
I can see a use for this with later additions but it is currently not used.

Reply:
You can also do req entries with conditions, so you could have the req active only when the skeleton mesh is present. (Written on mobile, so sorry for terseness and lack of examples. :))

Reply:
hmm I tried, oh well, guess I did something wrong.

Reply:
Okay, so now I'm not on mobile. ;)
```yaml
    req:
      - name: *FNIS
```
Won't work because `&FNIS` already includes the `name`. What that above essentially is saying is:
```yaml
    req:
      - name:
        - name: 'tools/GenerateFNIS_for_Users/GenerateFNISforUsers.exe'
          display: '[Fores New Idles in Skyrim SE](https://www.nexusmods.com/skyrimspecialedition/mods/3038)'
```
which is not valid LOOT masterlist syntax. :)

Using
```yaml
    req:
      - *FNIS
```
Or
```yaml
    req:
      - <<: *FNIS
        condition: 'file("meshes/actors/character/character assets/skeleton.hkx")'
```
(Or even
```yaml
    req:
      - <<: *FNIS
        display: '[Fores New Idles in Skyrim SE](https://www.nexusmods.com/skyrimspecialedition/mods/3038) (when using this specific skeleton)'
        condition: 'file("meshes/actors/character/character assets/skeleton.hkx")'
```
) should work. :)

Reply:
well crap, thanks :)